### ISSUE TITLE
Wrong runtime version dependency in 5.0 preview 3 build

### ISSUE THREAD

**Issue by mthalman** [dotnet/windowsdesktop#705]
In the 5.0.100-preview.3.20202.4 build of the SDK, Microsoft.WindowsDesktop.App depends on version 5.0.0-preview.3.20201.7 of Microsoft.NETCore.App.  But the version of Microsoft.NETCore.App contained in the SDK package is 5.0.0-preview.3.20169.1.

This will cause the following error:
```
It was not possible to find any compatible framework version

The framework 'Microsoft.NETCore.App', version '5.0.0-preview.3.20201.7' was not found.

The following frameworks were found:

5.0.0-preview.3.20169.1 at [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
```

**Comment by mmitche** [comment]
@mthalman This is because the latest build of the sdk is failing because of TLS1.2 enforcement. The latest check-ins into the SDK have this new runtime but it is failing to build: https://github.com/dotnet/installer/issues/7050

**Comment by merriemcgaw** [comment]
@mthalman , @mmitche - is this resolved with the latest Preview?

**Comment by mthalman** [comment]
Yes, this was fixed in 5.0.100-preview.3.20208.17.  Closing.
